docs/building-on-etherlink/endpoint-support.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Ethereum endpoint support'
+---
+
+Etherlink nodes use [Geth](https://geth.ethereum.org/) to provide access to both standard Ethereum RPC endpoints and Geth-specific RPC endpoints.
+These tables list these endpoints and whether Etherlink supports them:
+
+## Standard Ethereum endpoints
+
+For information about these endpoints, see [JSON-RPC API](https://ethereum.org/en/developers/docs/apis/json-rpc) and the [Ethereum JSON-RPC Specification](https://ethereum.github.io/execution-apis/api-documentation/) in the Ethereum documentation.
+
+Endpoint | Supported | Notes
+--- | --- | ---
+`debug_getBadBlocks` | No |
+`debug_getRawBlock` | No |
+`debug_getRawHeader` | No |
+`debug_getRawReceipts` | No |
+`debug_getRawTransaction` | No |
+`engine_exchangeCapabilities` | No |
+`engine_exchangeTransitionConfigurationV1` | No |
+`engine_forkchoiceUpdatedV1` | No |
+`engine_forkchoiceUpdatedV2` | No |
+`engine_forkchoiceUpdatedV3` | No |
+`engine_getPayloadBodiesByHashV1` | No |
+`engine_getPayloadBodiesByHashV2` | No |
+`engine_getPayloadBodiesByRangeV1` | No |
+`engine_getPayloadBodiesByRangeV2` | No |
+`engine_getPayloadV1` | No |
+`engine_getPayloadV2` | No |
+`engine_getPayloadV3` | No |
+`engine_getPayloadV4` | No |
+`engine_newPayloadV1` | No |
+`engine_newPayloadV2` | No |
+`engine_newPayloadV3` | No |
+`engine_newPayloadV4` | No |
+`eth_accounts` | Yes | This endpoint always returns an empty list because the Etherlink EVM node does not manage any addresses
+`eth_blobBaseFee` | No |
+`eth_blockNumber` | Yes |
+`eth_call` | Yes | Etherlink nodes use the standard Ethereum version of the `eth_call` endpoint instead of the Geth version; Ethereum does not support the state override set parameter
```

Comment:
```suggestion
`eth_call` | Yes | Etherlink nodes use the standard Ethereum version of the `eth_call` endpoint instead of the Geth version; we do not support the state override set parameter
```

Or am I interpreting this comment incorrectly?

Reply:
Yes, I'll change this to "Etherlink does not support..."